### ISSUE TITLE
Y24-383 - Cherrypicking improvements (minimum volume)

### ISSUE THREAD

**Issue by KatyTaylor** [sanger/sequencescape#4413]
**User story**
As SSR and Sample Management stakeholders, I would like the following behaviour changes to the cherrypicking pipeline.

**Who are the primary contacts for this story**
Stephen, Katy

**Who is the nominated tester for UAT**
Liz C, Lynda

**Acceptance criteria**
To be considered successful the solution must allow:
- [x] If required volume < robot minimum picking volume, round up to the robot minimum picking volume*
- [ ] If source volume < robot minimum picking volume, warn the user (but allow them to continue)
- [x] Change the robot minimum picking volume to only apply to sample (V) not buffer (B), in amount and concentration picks**
- [x] Change the final destination volume to equal the actual picked volume (V+B), in amount and concentration picks
- [x] Prevent the user from entering negative amounts when cherrypicking, at the form level.
- [x] Rename fields (in UI) as follows (for clearer user experience):
    - [x] Volume Required (µl) --> Target Volume Required (µl)
    - [x] Concentration Required (µl) --> Target Concentration Required (µl)
    - [x] Minimum Volume (µ/l) --> Minimum Target Volume (µl)
    - [x] Maximum Volume (µ/l) --> Maximum Target Volume (µl)
    - [x] Volume (µl) --> Volume to pick (µl)

**Dependencies**
This story is blocked by the following dependencies:
- https://github.com/sanger/sequencescape/issues/4412

**References**
This story has a non-blocking relationship with:
- https://github.com/sanger/sequencescape/issues/4410
- https://github.com/sanger/sequencescape/issues/4411

**Additional Context**

Screenshot of where the renaming of fields will appear:
<img width="1238" alt="Image" src="https://github.com/user-attachments/assets/93e2f95b-071b-4be3-b4cc-c5e36a610853" />

*this is because liquid handlers struggle to pick up very small volumes. If you want it to pick up 1μl, the robot is more successful at picking it up if it is programmed to pick up 5μl (because it builds up more negative pressure). Based on experiments Scott T (R&D) did many years ago.

**this is because the viscosity of the liquid is important. We don't know what the viscosity of the sample will be, so we make the safer assumption that it will be high (and take precautions - minimum picking volume). We know the buffer will have low viscosity so the robot can pick up small volumes more easily.

**Comment by SujitDey2022** [comment]
@Stan1520 @KatyTaylor adding a On Hold Tag based on Liz's email where she mentioned they will looking at the Instrument Issue Management process and revert on the priority for the following story after discussion with the Senior Managers.

**Comment by KatyTaylor** [comment]
Confirmed 1/4/25 with Ruth and Nat that this story can be progressed separately from the impact assessment - ready to be prioritised. FYI @SujitDey2022

**Comment by KatyTaylor** [comment]
Reopening this issue because I believe it was never released. Closing the pull request but there is an active branch here - [revert-5080-revert-4939-y24-383-cherrypicking-improvements](https://github.com/sanger/sequencescape/tree/revert-5080-revert-4939-y24-383-cherrypicking-improvements) - in case we ever come back to it.

Last update was from Liz Cook 27/8/25, email thread "Y24-383 - Cherry-picking UAT"
She wasn't able to prioritise the UAT at that time.
